[SSL] Update faq.md (#11949)
cloudflare/cloudflare-docs: content/ssl/troubleshooting/faq.md

## Does Cloudflare SSL support Internationalized Domain Names (IDN)?

> The double byte / IDN / punycode domains support for Cloudflare Edge certificates depends on the [Certificate Autority (CA)](/ssl/reference/certificate-authorities).

Reviewer: ```suggestion
The double byte / IDN / punycode domains support for Cloudflare edge certificates depends on the [certificate authority (CA)](/ssl/reference/certificate-authorities).
```

> Google Trust Services do not support punycode domains as mentionned in the [Certificate Authorities limitations](/ssl/reference/certificate-authorities/#limitations-1).

Reviewer: ```suggestion
Google Trust Services does not support punycode domains as mentioned in the [certificate authorities limitations](/ssl/reference/certificate-authorities/#limitations-1).
```

For a safer alternative, use [Certificate Transparency Monitoring](/ssl/edge-certificates/additional-options/certificate-transparency-monitoring/).

> Please refer to our [Certificate pinning](/ssl/reference/certificate-pinning/) article for more details.

Reviewer: ```suggestion
Refer to [Certificate pinning](/ssl/reference/certificate-pinning/) for more details.
```
